src/src/algo/hillel/InsertionSort.java

```diff
@@ -0,0 +1,25 @@
+package algo.hillel;
+
+import java.util.Arrays;
+
+public class InsertionSort {
+    public static void main(String[] args) {
+        var ints = ArrayUtils.doArray(100);
+        System.out.println(Arrays.toString(ints));
+        var sort = insertSort(ints);
+        System.out.println(Arrays.toString(ints));
+    }
+    private  static  int[] insertSort(int[] ints){
```

Comment:
This algo is not working! Fix it please.
I've run it with these numbers and received the following result:
`[14, 4, 14, 0, 13, 14, 13, 11, 0, 0, 1, 4, 10, 13, 13]
[0, 0, 0, 1, 4, 4, 10, 11, 13, 13, 13, 13, 14, 14, 13]`